packages/components/organisms/f-checkout/test-utils/component-objects/f-checkout-selectors.js

```diff
@@ -54,7 +54,7 @@ export const FIELDS = {
         typeError: '[data-test-id="error-address-postcode-type-error"]'
     },
     userNote: {
-        input: '[data-test-id="formfield-textarea"]',
+        input: '[data-test-id="formfield-Note-textarea"]',
```

Comment:
Super NIT – can `Note` be lowercase, or does it get auto-populated somewhere (could be one to update in a separate PR).

Reply:
It can be updated, It's auto-populated but it's just because we've put `name=Note` instead of `name=note` so I'll change it :)